packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
```

Comment:
Perhaps add a new `logging.dart` file?

Reply:
👍 Works for me (and that'd be my preference actually).

@Hixie: do you care either way?